packages/react-app/src/components/manage/substake.js

```diff
@@ -103,7 +103,7 @@ export const SubStakeList = (props) => {
                     Start
                 </Col>
                 <Col xs={12} sm={3} className="d-flex justify-content-start">
-                    End
+                    Unlock
```

Comment:
👍🏻 Much more clear (if it's accurate!)

Reply:
Yeah I'm thinking I will re-frame all of these as "unlock date" including on stake creation and extend.  Right?

Reply:
Right now it's a bit confusing between "end" and unlock.  People think they can onlock on their end date but that's really the start date of the last period.

Reply:
I agree.  Perhaps "Lock" and "Unlock" respectively.

Reply:
It also might be worth including a tooltip mentioning that the unlock date is dependent on wind down.